versions/3.0.2.md

```diff
@@ -1363,7 +1363,7 @@ Each Media Type Object provides schema and examples for the media type identifie
 ##### Fixed Fields
 Field Name | Type | Description
 ---|:---:|---
-<a name="mediaTypeSchema"></a>schema | [Schema Object](#schemaObject) \| [Reference Object](#referenceObject) | The schema defining the type used for the request body.
+<a name="mediaTypeSchema"></a>schema | [Schema Object](#schemaObject) \| [Reference Object](#referenceObject) | The schema defining the type used for the request body (or response body, if this is part of a response).
```

Comment:
Note that Media Type Object is also used in Parameter Object, in cases where the parameter is specified by the `content` map, as opposed to `schema` and `style`.  The `content` property is a Map[`string`, Media Type Object].

Maybe it would be simpler to say:

 "The schema defining the content of the request, response, or parameter."

Reply:
Good catch, sir.  :)

Reply:
I like that wording a lot better, actually, because the old wording:

> The schema defining the type used

Implies that the schema uniquely identifies the "type" of a request, response, or parameter, but since a schema can validate multiple types, it doesn't really.